src/mainboard/starfive/visionfive2/bt0/Cargo.toml

```diff
@@ -16,3 +16,12 @@ spin = "0.9"
 
 layoutflash = { path = "../../../../lib/layoutflash" }
 log = { path = "../../../../lib/log" }
+
+[dependencies.jh71xx-pac]
+git = "https://github.com/rmsyn/jh71xx-pac"
```

Comment:
The `jh71xx-pac` dependency is currently pinned to the `HEAD` of the git repository. If/when changes in this PR are approved, I'll push the next version of the crate to crates.io, and change the dependency entry here.